src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -37,6 +37,11 @@ import VariantCountCache from "./VariantCountCache";
 
 type PageMode = 'patient' | 'sample';
 
+class OncokbDefault implements IOncoKbData {

```

Comment:
@adamabeshouse you're right.  this could be a constant because immutable will clone it.  I looked in mobxpromise to see if default was returned as immutable, but in fact, as you point out, it's done in remoteData